app/Models/IndicatorValue.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\Traits\HasUploadFields;
+use Backpack\CRUD\app\Models\Traits\CrudTrait;
+use Illuminate\Database\Eloquent\Model;
+
+class IndicatorValue extends Model
+{
+    use CrudTrait, HasUploadFields;
+
+    /*
+    |--------------------------------------------------------------------------
+    | GLOBAL VARIABLES
+    |--------------------------------------------------------------------------
+    */
+
+    protected $table = 'indicator_values';
+    // protected $primaryKey = 'id';
+    // public $timestamps = false;
+    protected $guarded = ['id'];
+    // protected $fillable = [];
+    // protected $hidden = [];
+    // protected $dates = [];
+    protected $casts = ['disaggregation_id' => 'array', 'file_source' => 'array' ];
+
+    /*
+    |--------------------------------------------------------------------------
+    | FUNCTIONS
+    |--------------------------------------------------------------------------
+    */
+
+    /*
+    |--------------------------------------------------------------------------
+    | RELATIONS
+    |--------------------------------------------------------------------------
+    */
+    public function indicator_status()
+    {
+        return $this->hasMany('App\Models\IndicatorStatus', 'indicator_status_id');
+    }
```

Comment:
Same as above -  not sure the relationship between IndicatorStatus and IndicatorValue is the right way round.